index.html

```diff
@@ -5,8 +5,44 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Henry Cadena Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <h1>Henry Cadena Portfolio Mobile First</h1>
+    <header class="container-header">
+      <a href="#">
+        <h3 class="logo">Henry Cadena</h3>
+      </a>
+      <nav class="container-burger">
+        <label class="burger1"></label>
+        <label class="burger2"></label>
+        <label class="burger3"></label>
+      </nav>
+    </header>
+    <main class="container-main">
+      <img class="hero-image" src="assets/background-mobile-version.svg" alt="background">
```

Comment:
- [ ] As the previous reviewer stated, you should use a background image here, instead of the **_img_** tag. Because If an image is not part of the content (aesthetic reasons 🎴 ) instead use CSS **background-image**. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. 😄 
You can achieve that this way 👇🏽 👇🏽 

```sh
. container-main {
  background-image: url(assets/background-mobile-version.svg);
  // Your other rules here
  }
 ```